### PR TITLE
First draft of IPFS Weekly #73

### PR DESCRIPTION
**Pull request by jennwrites**
Still experimenting with format for distribution through MailChimp, but here is the bulk of the content so far. 

To be published Tuesday, January 14 after 12pm Central time. 

If folks have dweb event recommendations, I'm listening 👂

### CODE REVIEW

**Comment by serejandmyself** [comment]
I would like to propose some news for "IPFS in the wild" section or "releases" section. 

TL;DR 
Cyber is a consensus computer on top of IPFS that uses IPFS hashes to create cyberlinks. In other words, it is a decentralized google built with the help of IPFS.

We have made a release of our final (we hope so) testnet: [Euler-5](https://github.com/cybercongress/cyberd/releases). It is an incentivized testnet that allows launching validator nodes, searching for content on IPFS and most importantly create cyberlinks (semantically link IPFS hashes to create a semantic core). 
An alpha version of the search UI can be found on [cyber.page](https://cyber.page/). 
There is also raw footage of the launch [video](https://www.youtube.com/watch?v=RS6w6wiAag4).

**Comment by jennwrites** [comment]
@serejandmyself I don’t currently have a section for releases, so I will include your project’s release in next week’s issue under a projects-focused section.
